BankDefault.lua

```diff
@@ -21,15 +27,14 @@ function BankDefault.OnAddOnLoaded(event, addonName)
 end
 
 function BankDefault.OnBankOpen(event, bankBag)
-    local bankScene = SCENE_MANAGER:GetScene("bank")
-    bankScene:RegisterCallback("StateChange", function(oldState, newState)
-        if newState == SCENE_SHOWING then
-            selectDeposit()
-        end
-    end)
+    --Do not call the scene callback function registration with EACH bank open!
+    --It's enopugh to do this once after addon init.
```

Comment:
You wouldn't need this event anymore, currently, as the scene callback will recognize ALL bank scene changes. Each time the bank scene is shown -> your function BankDefault.selectDeposit() will be called

Reply:
Thank you for the feedback. Your comments are very helpful.